elements/lisk-p2p/src/peer/base.ts

```diff
@@ -145,6 +156,11 @@ export class Peer extends EventEmitter {
 		this._wsPort = peerInfo.wsPort;
 		this._id = constructPeerId(this._ipAddress, this._wsPort);
 		this._height = peerInfo.height ? peerInfo.height : 0;
+		this._reputation = DEFAULT_REPUTATION_SCORE;
+		this._callCounter = 0;
```

Comment:
Where doest the `_callCounter` get incremented?

Also I was thinking that this could be more advanced; we could count per peer **and** per peer event/procedure type. So the _callCounter could be a `Map` which keeps track of counts for each different kind of event.
But we have to be sure to cleanup this map correctly.